recipe/build.sh

```diff
@@ -13,6 +13,9 @@ set -x -e
 INCLUDE_PATH="${PREFIX}/include"
 LIBRARY_PATH="${PREFIX}/lib"
 
+# Always build PIC code for easier static linking
+CXXFLAGS="${CXXFLAGS} -O3 -fPIC"
```

Comment:
Is `O3` safe? I think I haven't seen optimization options added to any conda package. Or is it needed by `fPIC`?